deepchem/models/torch_models/layers.py

```diff
@@ -23,6 +23,173 @@
   from collections import Sequence as SequenceCollection
 
 
+class LinearModule(nn.Module):
```

Comment:
We should double check there isn't already a linear layer.

Need to add docs and should add to the layers.rst file